modules/cloud-config-container/simple-nva/README.md

```diff
@@ -62,16 +62,95 @@ module "vm" {
 }
 # tftest modules=1 resources=1
 ```
+
+### Example with advanced routing capabilities
+
+Below a sample terraform example for bootstrapping a simple-nva powered by [COS](https://cloud.google.com/container-optimized-os/docs) and running [FRRouting](https://frrouting.org/) container.
+Please find below s sample frr.conf file based on documentation available at the following [link](https://docs.frrouting.org/en/latest/basic.html) for hosting a BGP service with ASN 65001 on FRR container establishing a BGP session with a remote neighbor with IP address 10.128.0.2 and ASN 65002.
```

Comment:
s --> a

Reply:
on documentation --> on the documentation

Reply:
at the following --> [here](...)

Reply:
done!